dev/NavigationView/NavigationView.cpp

```diff
@@ -2197,6 +2177,90 @@ void NavigationView::HandleKeyEventForNavigationViewItem(const winrt::Navigation
             args.Handled(true);
             KeyboardFocusLastItemFromItem(nvi);
             break;
+        case winrt::VirtualKey::Down:
```

Comment:
>Down [](start = 32, length = 4)

This is switching on args.Key() - meaning that "Down" here will trigger for DPadDown and LeftThumbstickDown too - I want to make sure you intend this given the name of the function including "Keyboard".  Make sure to test behavior here with an xbox controller (on desktop is fine, no need to use an xbox console)